projects/destinations_similarity/README.md

```diff
@@ -0,0 +1,92 @@
+# Destinations Similarity
+
+### Short description
+
+Similar destination search.
+
+### Problem statement
+
+Kinzinho is an adventurous dog who wants to know all about the destinations he could go to. To do so, Kinzinho decided to extract public data from Wikipedia and Wikivoyager to get to know them all! But now he realized it's too much and wants some recommendations based on where he's traveled before. Can we help him?
+
+Tip: of course! Everything is 'paw-sible' when you are a dog! :D
+
+![kinzinho Big Dog](docs/images/kinzinhoBigDog.png)
+
+
+### Solution implementation
+
+The solution to the problem was to extract the public database of Brazilian cities from Wikidata and model the relevant characteristics of cities to build a unique representation of each city. From there, we were able to determine the similarities by calculating the distances between the
+vector representations.
+
+The system's workflow was implemented in Flyte and is shown below:
+
+![Solution Workflow](docs/images/SolutionDiagram.png)
+
+### Detailed solution
+
+**Objective**: To help Kinzinho define his next travel destination, we seek to find other cities similar to the last travel destination he liked the most.
+
+**Strategy Solution**: To make a good evaluation between the cities, we chose to make a vector representation of each city in Brazil, encoding general information about each city such as its history, its geography, its climate and its tourist attractions. We chose this strategy because with a vector representation of the city we were able to apply similarity calculation operations between cities, considering various information about them.
+
+**Input data**: For our solution we use the following data from Wikipedia PT: summary, history, geography, climate; from Wikivoyage EN: summary, "See" section, "Do" section.
+
+**Preprocessing**: To process the data and extract only important information, we apply a series of pre-processing to clean unnecessary information and homogenize the texts.
+
+**Model**: To generate the best vector representation of each city's features, we used a pre-trained state-of-the-art model based on Transformers (BERTimbau). As a vector representation of each feature of the city, we use the output of the last layer of the BERTimbau language model. The vector representation of each city is generated from the average of the vectors of its features.
```

Comment:
I think the README needs to include a link to https://github.com/facebookresearch/faiss since you're using it to implement KNN. What do you think?